charts/osm/templates/azure-policy-osm-label.yml

```diff
@@ -0,0 +1,123 @@
+# 
+# The job in this YAML file tries to add 'openservicemesh.io/ignore' label to 'kube-system' and 'azure-arc' namespaces.
+# The job is executed in helm pre-install phase of helm life cycle.
+#
+# For namespaces with 'openserviemesh.io/ignore' label...
+#     1. Kubernetes admission requests with violations are not denied by Gatekeeper.
+#     2. Auditing results are still available.
+#
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: azure-policy-osm-label-account
+  namespace: kube-system
```

Comment:
you can get this namespace from values.yaml once Sneha adds it in the other PR

Reply:
Does the "arc-osm-system" namespace that Sneha is adding impact anything in the osm-label job? Because regardless we are still adding the openservicemesh.io/ignore label only to the azure-arc and kube-system namespaces?